pkg/storage/pebble.go

```diff
@@ -129,6 +130,32 @@ var MinCapacityForBulkIngest = settings.RegisterFloatSetting(
 	0.05,
 )
 
+// readaheadModeInformed controls the pebble.ReadaheadConfig.Informed setting.
+var readaheadModeInformed = settings.RegisterEnumSetting(
+	settings.ApplicationLevel, // used by temp storage as well
+	"storage.readahead_mode.informed",
+	"the readahead mode for operations which are known to read through large chunks of data (0=off, 1=SYS_READAHEAD, 2=FADV_SEQUENTIAL)",
```

Comment:
is the 0,1,2 part of this description still valid now that we have an enum type? alternatively, we could add a description here for stuff like what sys-readahead means. And this is really a nit, but I'd say pebble-readahead instead of sys-readahead as pebble drives the blocks to read ahead, while sys-readahead sounds more like fadvise sequential.